db-migrations/src/main/resources/db/migration/V2__add_deleted_flag.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE expenses
+  ADD deleted BOOLEAN;
```

Comment:
Sollten wir das vielleicht NOT NULL machen und alle anderen explizit auf false setzen?